src/main/webapp/index.html

```diff
@@ -2,15 +2,20 @@
 <html lang="en-US">
   <head>
     <meta charset="UTF-8" />
-    <title>Reddit Dashboard</title>
+    <title>Youtube Comment Dashboard</title>
     <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="addThreads()">
     <div id="content">
-      <h1>Reddit Dashboard</h1>
-      <button onclick="addThreads()" type="button">Get Response</button>
+      <h1>Youtube Comment Dashboard</h1>
       <div id="thread-container"></div>
+      <button type="button" id="previous" onclick="previousPage()" value="Previous">Previous</button>
```

Comment:
I see that #previousPage, #numberPage, and #nextPage don't exist yet - maybe add empty functions with these names to the js with TODOs inside, or add a TODO comment here noting to implement them